src/main/java/uk/gov/companieshouse/ocr/api/image/extracttext/OcrClientRequest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.ocr.api.image.extracttext;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A Pojo used in a Web Request sent by a client to this microservice
+ */
+public class OcrClientRequest {
+
+    @JsonProperty("app_id")
+    private String applicationID;
```

Comment:
all ID should be Id as per code guidelines
![image](https://user-images.githubusercontent.com/55089213/123998663-9c5e1f00-d9c9-11eb-80d4-febb2b43b43b.png)